tests/migration_test.go

```diff
@@ -486,12 +487,13 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 					// check VMI, confirm migration state
 					confirmVMIPostMigration(vmi, migrationUID)
 
-					By("Check if Migrated VMI has updated PodIP")
+					By("Check if Migrated VMI has updated IP and IPs fields")
 					Eventually(func() bool {
 						newvmi, err := virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Get(vmi.Name, &metav1.GetOptions{})
 						Expect(err).ToNot(HaveOccurred(), "Should successfully get new VMI")
-						vmiPod := tests.GetRunningPodByVirtualMachineInstance(newvmi, tests.NamespaceTestDefault)
-						return newvmi.Status.Interfaces[0].IP == vmiPod.Status.PodIP
+						vmiPod := tests.GetRunningPodByVirtualMachineInstance(newvmi, newvmi.Namespace)
+						err = libvmi.ValidateVMIandPodIPMatch(newvmi, vmiPod)
+						return err == nil
```

Comment:
Please change this `Eventually` function to return error instead of bool, with meaningful error messages

Reply:
done